TAGradingServer/account/admin-rubric.php

```diff
@@ -527,7 +527,7 @@ function calculatePercentageTotal() {
         var ec = 0;
         $('select.points').each(function(){
             var elem = $(this).attr('name').replace('point','ec');
-            if (!$('[name="'+elem+'"]').prop('checked') == true) {
+            if (!$('[name="'+elem+'"]').is(':checked') == true) {
```

Comment:
Here's a jsfiddle that shows an example of why I'm doing this: https://jsfiddle.net/g850h7mh/

Basically, a disabled checkbox doesn't send a value when the form is submitted to the php page, so I use a hidden value to take it's place. Thus $('[name="ec-0-2"]') selects the hidden field which returns false for ```.prop('checked')``` but true for ```.is(':checked')```